images/runtime/python/generateDockerfiles.sh

```diff
@@ -21,9 +21,9 @@ declare -r ORYX_PYTHON_IMAGE_BASE_PLACEHOLDER="%BASE_TAG%"
 
 source "$PYTHON_VERSIONS_PATH"
 
-declare -r PYTHON_BULLSEYE_VERSION_ARRAY=($PYTHON310_VERSION)
+declare -r PYTHON_BULLSEYE_VERSION_ARRAY=($PYTHON37_VERSION $PYTHON310_VERSION)
 declare -r PYTHON_BUSTER_VERSION_ARRAY=($PYTHON39_VERSION)
-declare -r PYTHON_STRETCH_VERSION_ARRAY=($PYTHON27_VERSION $PYTHON36_VERSION $PYTHON37_VERSION $PYTHON38_VERSION)
+declare -r PYTHON_STRETCH_VERSION_ARRAY=($PYTHON27_VERSION $PYTHON36_VERSION $PYTHON38_VERSION)
```

Comment:
As discussed separately on Teams, could you please confirm that Python 3.8 is indeed using Stretch? If that's the case, could you please let me know how easy it would be to push it to Bullseye as well?

Reply:
Currently python 2.7 3.6 and 3.8 are still in stretch, it won't be much efforts for us to update it to bullseye based, do we want to proceed and migrate those as well?

Reply:
Thanks for confirming! For 2.7 and 3.6, we don't need to take care of them because they are already end of life. However, 3.8 will still be supported even after June 30 (Stretch EOL), so we need to move it to Bullseye. Could you please migrate this one as well?

Reply:
Ok, will update it.